src/nodes.coffee

```diff
@@ -2872,6 +2894,23 @@ exports.ExecutableClassBody = class ExecutableClassBody extends Base
       assign
     compact result
 
+exports.ClassProperty = class ClassProperty extends Base
+  constructor: ({@name, @isStatic, @staticClassName, @value, @operatorToken}) ->
+    super()
+
+  children: ['name', 'value', 'staticClassName']
+
+  isStatement: YES
+
+  astProperties: (o) ->
+    return
+      key: @name.ast o, LEVEL_LIST
+      value: @value.ast o, LEVEL_LIST
+      static: !!@isStatic
+      computed: no
+      operator: @operatorToken?.value ? '='
```

Comment:
`operator` and `staticClassName` are Coffeescript-specific extensions to the Babel `ClassProperty` AST node type (and similarly on `ClassMethod` below)